corehq/apps/sms/migrations/0042_auto_20200622_1243.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2020-06-22 12:43
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('sms', '0041_email'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='InfobipBackend',
```

Comment:
these were already added in https://github.com/dimagi/commcare-hq/pull/27851/files so you might need to update this here.